src/main.rs

```diff
@@ -0,0 +1,86 @@
+use actions_github::context::get_context;
+use actions_github::core::{get_input, set_output};
+use actions_github::logger;
+use anyhow::Result;
+use octocrab::Octocrab;
+
+#[tokio::main]
+async fn main() {
+    let ctx = match get_context() {
+        Ok(context) => context,
+        Err(error) => panic!("{}", error),
+    };
+    let org: String = ctx.repo.owner;
+
+    let team_name: String = get_variable("team", true);
+
+    let token: String = get_variable("ACCESS_TOKEN", true);
+
+    let crab = Octocrab::builder().personal_token(token).build();
+    octocrab::initialise(crab.unwrap());
+
+    let team = fetch_team(org, &team_name).await.unwrap();
+
+    if team.is_empty() {
+        panic!(
+            "No users where found while searching for the team {}",
+            &team_name
+        );
+    }
+
+    logger::debug_log(format!("Obtained data from {} users", team.len()).as_str());
+    if let Err(err) = set_output(
+        "usernames",
+        team.clone()
+            .iter()
+            .map(|member| member.username.clone())
+            .collect::<Vec<String>>()
+            .join(",")
+            .as_str(),
+    ) {
+        panic!("{:#?}", err);
+    }
```

Comment:
same here, `if let Err(...) {panic!(...)}` can be replaced with `.unwrap()` or `.expect("message")`

```rust
set_output(
    "usernames",
    team.clone()
        .iter()
        .map(|member| member.username.clone())
        .collect::<Vec<String>>()
        .join(",")
        .as_str(),
).expect("set_output failed");
```